Sources/StreamChatSwiftUI/ChatChannel/MessageList/MessageView.swift

```diff
@@ -275,30 +276,16 @@ public struct LinkDetectionTextView: View {
     
     public var body: some View {
         Group {
-            if let displayedText {
-                Text(displayedText)
-            } else {
-                Text(message.adjustedText)
-            }
```

Comment:
In case of `LinkDetectionTextView`, I'm considering passing a new property, maybe `originalText: String?` instead of the whole view model, so that it requires less changes. Still need to check if it would work.